.github/workflows/ci-bump-dask.yml

```diff
@@ -0,0 +1,43 @@
+name: Check for new dask version on conda-forge
+
+on:
+  schedule:
+    - cron: "36 * * * *"
```

Comment:
I suppose in theory these two bump actions could result in merge conflicts, but only if there is a release of the two very close to each other. I'm not too worried about it in practice

Reply:
Hmm I wonder if `ci-bump-dask.yml` and `ci-bump-coiled.yml` can be merged into a single workflow. This is definitely not a blocking comment, just a passing thought

Reply:
My initial pass at this actually did that, but it started to get hairy with different task IDs, and no real reduction in lines-of-code, so I cut bait

Reply:
👍

Reply:
Apparently I have to take another look at this :/